src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  const discount7Days = 50;
+  const discount3Days = 20;
```

Comment:
do not attach var names to other values (days)
use abstract names `smallDiscount` `largeDiscount`